release.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+# Usage: $ ./prepare_for_release.py <type>
+# type - [major | minor | patch]. Default is patch.
+
+get_current_package_version()
+{
+  echo $(python ./setup.py --version)
+}
+
+set_next_package_version() {
+  bumpversion --current-version $1 --commit --tag --tag-name {new_version} $2 ./setup.py
```

Comment:
should `{new_version}` be `${new_version}`? also when does `${new_version}` get set?

Reply:
I think `{new_version}` is a regex for the tag name. It's not related to the bash environment.
From the help file of `bumpversion`:
```
  --tag-name TAG_NAME   Tag name (only works with --tag) (default:
                        v{new_version})
```
SInce we don't want that `v` there, I just adjusted the value using the variable they specified.

Reply:
ah ok, -- might be worth a comment in there to not touch `{new_version}` because `bumpversion` requires it.

Reply:
Added :)